account_commission_listing_fee/views/commission_settlement_views.xml

```diff
@@ -9,6 +9,18 @@
                 <field name="event_ids" widget="many2many_tags" />
                 <field name="number_of_items" />
             </xpath>
+            <xpath
+                expr="//field[@name='line_ids']/tree/field[@name='date']"
+                position="after"
+            >
+                <field name="product_id" />
+            </xpath>
+            <xpath
+                expr="//field[@name='line_ids']/tree/field[@name='commission_id']"
+                position="attributes"
+            >
+                <attribute name="groups">account.group_account_invoice</attribute>
```

Comment:
Should not we add for group_commission_manager and group_commission_user for the case of creating directly settlements?

Reply:
The point is to hide the field from auction users, so `base.group_user` should suffice.

Reply:
My goal was to hide that field from portal users. so I will just update it to base_group_user. Thanks for review.